RecoTauTag/TauTagTools/interface/ECALBounds.h

```diff
@@ -1,41 +1,48 @@
-#ifndef RecoTauTag_TauTagTools_ECALBounds_H
-#define RecoTauTag_TauTagTools_ECALBounds_H
+#ifndef GeomPropagators_ECALBounds_H
```

Comment:
please keep the include guards match the full name of the file: it's not a GeomPropagators package here